src/Sarif.FunctionalTests/UpdateBaselines.ps1

```diff
@@ -2,7 +2,7 @@
     [string]$ToolName
 )
 
-$utility = "$PSScriptRoot\..\..\bld\bin\AnyCPU_Release\net452\Sarif.Multitool.exe"
+$utility = "$PSScriptRoot\..\..\\bld\bin\Sarif.Multitool\AnyCPU_Release\net461\Sarif.Multitool.exe"
```

Comment:
Remove extra backslash "`..\\bld`".

Thanks for fixing; this wasn't even on my punch list.